src/main/java/net/minecraftforge/event/entity/player/AdvancementEvent.java

```diff
@@ -32,4 +34,99 @@ public Advancement getAdvancement()
     {
         return advancement;
     }
+
+    /**
+     * This event is fired when a player earns an advancement/recipe.
+     * <br>
+     * This event is not {@link net.minecraftforge.eventbus.api.Cancelable}.<br>
+     * <br>
+     * This event does not have a result. {@link HasResult}<br>
+     * <br>
+     * This event is fired on the {@link MinecraftForge#EVENT_BUS}.
+     */
```

Comment:
Here's some cleaned up javadocs, using the (as of yet uncodified) format for event javadocs:
```suggestion
    /**
     * Fired when the player earns an advancement. An advancement is earned once its requirements are complete.
     *
     * <p>Note that advancements may be hidden from the player or used in background mechanics, such as recipe
     * advancements for unlocking recipes in the recipe book.</p>
     *
     * <p>This event is not {@linkplain Cancelable cancellable}, and does not {@linkplain HasResult have a result}.</p>
     *
     * <p>This event is fired on the {@linkplain MinecraftForge#EVENT_BUS main Forge event bus},
     * only on the {@linkplain LogicalSide#SERVER logical server}.</p>
     *
     * @see AdvancementProgress#isDone()
     */
```